src/lib/bootloader/grub2bls.rb

```diff
@@ -155,7 +157,7 @@ def packages
       res = super
       res << ("grub2-" + grub2bls_architecture + "-efi-bls")
       res << "sdbootutil"
-      res << "shim" if secure_boot
+      res << "shim"
```

Comment:
why this change is needed? I think there are architectures where shim is no available, but as it is grub2-bls architectures are already limited, os not sure.